docs/README.md

```diff
@@ -1,58 +1,82 @@
-# Scylla documentation
+# ScyllaDB Documentation
 
-This is the repository for [Scylla documentation site](http://docs.scylladb.com/)
+This repository contains the source files for ScyllaDB Open Source documentation.
 
-To report an issue with the documentation, please use GitHub issues.
+- The `dev` folder contains developer-oriented documentation related to the ScyllaDB code base. It is not published and is only available via GitHub.
+- All other folders and files contain user-oriented documentation related to ScyllaDB Open Source and are sources for [docs.scylladb.com](https://docs.scylladb.com/).
 
-This repository accepts GitHub pull requests.
+To report a documentation bug or suggest an improvement, open an issue in [GitHub issues](https://github.com/scylladb/scylla/issues) for this project.
 
-**Send pull requests to the master branch, not gh-pages.  gh-pages will be overwritten by deploy without warning.**
+To contribute to the documentation, open a GitHub pull request.
 
+## Key Guidelines for Contributors
 
-## Prerequisites
+- You need to fork this repository.
+- Work on your fork's task branches, not the `master` branch.
```

Comment:
```suggestion
```

I do not think we need this part, its not different than  any other project
we already have https://github.com/scylladb/scylla/blob/master/CONTRIBUTING.md

Reply:
I agree with Tzach - I don't think our own README needs to explain the basics of github. 
Some of your suggestions, like "work on your fork's task branches, not the master branch", I am not even sure what they mean. After all, ordinary users cannot touch Scylla's master branch anyway, and on their own fork they can call their branch whatever they want, including "master" if they wish.